roscala/src/main/scala/coop/rchain/rosette/Op.scala

```diff
@@ -4,59 +4,60 @@ sealed trait Op
 case class OpHalt() extends Op
 case class OpPush() extends Op
 case class OpPop() extends Op
-case class OpNargs(next: Int) extends Op
-case class OpAlloc(next: Int) extends Op
-case class OpPushAlloc(next: Int) extends Op
+case class OpNargs(n: Int) extends Op
+case class OpAlloc(n: Int) extends Op
+case class OpPushAlloc(n: Int) extends Op
 case class OpExtend(v: Int) extends Op
 case class OpOutstanding(pc: Int, n: Int) extends Op
 case class OpFork(pc: Int) extends Op
-case class OpXmitTag(u: Boolean, n: Boolean, m: Int, v: Int) extends Op
+case class OpXmitTag(unwind: Boolean, next: Boolean, nargs: Int, v: Int)
```

Comment:
`v` can be `lit` here. Also if you use `nargs` here, please also use `nargs` for `OpNargs`.